config/routes.rb

```diff
@@ -146,6 +146,13 @@
 
       scope module: :forum do
         resources :forums do
+          resources :topics do
+            post 'subscribe', on: :member
+            delete 'unsubscribe', on: :member
+            put 'locked', on: :member
+            put 'hidden', on: :member
```

Comment:
To me using a verb as action makes more sense, so as discussed I think `post lock/hide` or `post set_hidden/locked` would be better. what do you guys think ?